Makefile

```diff
@@ -16,7 +18,6 @@ install: build
 		mv ${BINARY} ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
 
 test:
-		go test -i $(TEST) || exit 1
```

Comment:
don't we still want this?

Reply:
The line below it looks to cover this deleted line and more. The `$(TEST)` is `go list ./... | grep -v 'vendor'` which outputs:
```
github.com/ngrok/terraform-provider-ngrok
github.com/ngrok/terraform-provider-ngrok/ngrok
github.com/ngrok/terraform-provider-ngrok/restapi
github.com/ngrok/terraform-provider-ngrok/transform
```
The `echo $(TEST) | xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4` line runs all of those happily. The deleted line just picks up the first directory and then exits. Running them both manually:

Old:
```
> make test  
go test -i $(go list ./... | grep -v 'vendor') || exit 1
?   	github.com/ngrok/terraform-provider-ngrok	[no test files]
```

New:
```
> make test
echo $(go list ./... | grep -v 'vendor') | xargs -t -n4 go test  -timeout=30s -parallel=4
go test '-timeout=30s' '-parallel=4' github.com/ngrok/terraform-provider-ngrok github.com/ngrok/terraform-provider-ngrok/ngrok github.com/ngrok/terraform-provider-ngrok/restapi github.com/ngrok/terraform-provider-ngrok/transform
?   	github.com/ngrok/terraform-provider-ngrok	[no test files]
ok  	github.com/ngrok/terraform-provider-ngrok/ngrok	(cached)
?   	github.com/ngrok/terraform-provider-ngrok/restapi	[no test files]
?   	github.com/ngrok/terraform-provider-ngrok/transform	[no test files]
```

Reply:
lg